install.sh

```diff
@@ -755,6 +755,11 @@ openrc_start() {
 
 # --- startup systemd or openrc service ---
 service_enable_and_start() {
+    if [ -f "/proc/cgroups" ] && [ "$(grep memory /proc/cgroups | awk '{ print $4 }')" -eq 0 ];
```

Comment:
This get much more portable if you use shell script internals instead of using `awk`:

```sh
if [ -f "/proc/cgroups" ] && [ "$(grep memory /proc/cgroups | while read -r n n n enabled; do echo $enabled; done)" -eq 0 ]; 
```